tests/fixtures/files/index.js

```diff
@@ -14,42 +13,6 @@ const all = [
 ];
 
 /*
- * Raw File Fixture
- *
- * @returns {array} - An array file objects as one receives from a multi-upload form
- */
-const raw = () => {
```

Comment:
These are now in *shared tests*. Added [in this pr](https://github.com/punchcard-cms/shared-tests/pull/21)